Rakefile

```diff
@@ -1,12 +1,10 @@
 require "bundler/gem_tasks"
-require 'rake/testtask'
+require 'rspec/core/rake_task'
 
-Rake::TestTask.new do |t|
```

Comment:
I think that this `Rake::TestTask` way is used to use `minitest`. It's not RSpec's way.